README.md

```diff
@@ -334,28 +334,28 @@ Will then start each component with the alloted defaults. ~3-6 mins to complete
 The defaults used are:
 
 - `0x13E23Ca74DE0206C56ebaE8D51b5622EFF1E9944` is the public address of the pre-funded account on the L1 network used 
-to deploy the Obscuro Management and the ERC20 contracts
+to deploy the Ten Management and the ERC20 contracts
 - `0x0654D8B60033144D567f25bF41baC1FB0D60F23B` is the public address of the pre-funded account on the L1 network used to 
-pay for Obscuro rollup transactions
+pay for Ten rollup transactions
 - `f52e5418e349dccdda29b6ac8b0abe6576bb7713886aa85abea6181ba731f9bb` is the private key of the pre-funded 
-account on the L1 network used to deploy the Obscuro Management and the ERC20 contracts
-- `0x0000000000000000000000000000000000000001` is the host id of the Obscuro node when starting in a local mode
-- `0xeDa66Cc53bd2f26896f6Ba6b736B1Ca325DE04eF` is the address of the Obscuro Management contract which is known a-priori as a nonce of 0 is used 
-- `0xC0370e0b5C1A41D447BDdA655079A1B977C71aA9` is the address of the ERC20 contract which represents OBX and is known a-priori as a nonce of 1 is used
+account on the L1 network used to deploy the Ten Management and the ERC20 contracts
+- `0x0000000000000000000000000000000000000001` is the host id of the Ten node when starting in a local mode
+- `0xeDa66Cc53bd2f26896f6Ba6b736B1Ca325DE04eF` is the address of the Ten Management contract which is known a-priori as a nonce of 0 is used 
+- `0xC0370e0b5C1A41D447BDdA655079A1B977C71aA9` is the address of the ERC20 contract which represents TEN and is known a-priori as a nonce of 1 is used
 - `0x51D43a3Ca257584E770B6188232b199E76B022A2` is the address of the ERC20 contract which represents ETH and is known a-priori as a nonce of 2 is used
 
-Once started Obscuroscan is available on `http://0.0.0.0:8098`.
+Once started Tenscan is available on `http://0.0.0.0:8098`.
 
 
 ### Building and running a local faucet
-Deploying and interacting with contracts on Obscuro requires ETH to be allocated to an account on the L2 via the faucet (or bridged across from the L1).
+Deploying and interacting with contracts on Ten requires ETH to be allocated to an account on the L2 via the faucet (or bridged across from the L1).
 The faucet image should be pulled (or optionally built from scratch), and the container built and started to allow requests to be made to it. 
 To pull the testnet faucet and run use;
 
 ```
 cd tools/faucet
-docker pull testnetobscuronet.azurecr.io/obscuronet/faucet_sepolia_testnet:latest  
-./container_run.sh -image testnetobscuronet.azurecr.io/obscuronet/faucet_sepolia_testnet:latest 
+docker pull testnettennet.azurecr.io/tennet/faucet_sepolia_testnet:latest  
```

Comment:
Is this what they are going to be? Be tempted to leave these sorts of references unchanged so they can be found when they get replaced in bulk.

Reply:
ok. this looks odd